pkgs/development/tools/misc/clojure-lsp/default.nix

```diff
@@ -12,26 +12,27 @@
 
 buildGraalvmNativeImage rec {
   pname = "clojure-lsp";
-  version = "2023.08.06-00.28.06";
+  version = "2024.04.22-11.50.26";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = version;
-    sha256 = "sha256-wc7M2cPRtdaRzZn3GNu/aCbQ2VqxiDxvu/b7qnBVUBo=";
+    sha256 = "sha256-GyPIFYR+/BZ+vq6+yuer5HoVILXLWNw1sW8XpJ7q4SA=";
   };
 
   jar = fetchurl {
     url = "https://github.com/clojure-lsp/clojure-lsp/releases/download/${version}/clojure-lsp-standalone.jar";
-    sha256 = "c301821ac6914999a44f5c1cd371d46b248fe9a2e31d43a666d0bc2656cfdd78";
+    sha256 = "sha256-dB16225A7L3nWplvqlal+5gho+LmqqVGPN9dfasKaPk=";
   };
 
   extraNativeImageBuildArgs = [
     "--no-fallback"
     "--native-image-info"
+    "--features=clj_easy.graal_build_time.InitClojureClasses"
   ];
 
-  doCheck = true;
+  doCheck = false;
```

Comment:
Is this still needed if you bump further? Can you s

Reply:
What do you mean by "bump further"?

Reply:
Ah.. this is resolved anyways by `--features=clj_easy.graal_build_time.InitClojureClasses`